docs-src/modules/ROOT/pages/connect/resolvers.adoc

```diff
@@ -452,6 +452,22 @@ Try it out:
 <div class="space"></div>
 ++++
 
+
+*Note:* If you are calling the parser directly, be sure to quote your query when using parameters like so:
+```clojure
+(parser {} (parser {} '[(::instruments {:sort :instrument/brand})])
```

Comment:
you repeated the parser

Reply:
ahh sorry about that! not sure what happened there..